js/src/css/beautifier.js

```diff
@@ -92,11 +92,15 @@ Beautifier.prototype.eatString = function(endChars) {
 Beautifier.prototype.eatWhitespace = function(allowAtLeastOneNewLine) {
   var result = whitespaceChar.test(this._input.peek());
   var isFirstNewLine = true;
-
+  var newlines = this._options.max_preserve_newlines;
```

Comment:
Delete `var isFirstNewLine = true;` and then 

```suggestion
  var newline_count = 0;
```